bin/run

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
```

Comment:
Ooh, okay. That makes sense. I have no experience with anything that's _not_ makefiles, so I'll give this a shot :)